entity-framework/core/index.md

```diff
@@ -1,39 +1,61 @@
 ---
 title: Overview of Entity Framework Core - EF Core
-description: General introductory overiew of Entity Framework Core
-author: rowanmiller
-ms.date: 10/27/2016
+description: General introductory overview of Entity Framework Core
+author: ajcvickers
+ms.date: 9/20/2020
 uid: core/index
 ---
 
 # Entity Framework Core
 
 Entity Framework (EF) Core is a lightweight, extensible, [open source](https://github.com/aspnet/EntityFrameworkCore) and cross-platform version of the popular Entity Framework data access technology.
 
-EF Core can serve as an object-relational mapper (O/RM), enabling .NET developers to work with a database using .NET objects, and eliminating the need for most of the data-access code they usually need to write.
+EF Core can serve as an object-relational mapper (O/RM), which:
+
+* Enables .NET developers to work with a database using .NET objects.
+* Eliminates the need for most of the data-access code that typically needs to be written.
 
 EF Core supports many database engines, see [Database Providers](xref:core/providers/index) for details.
 
-## The Model
+## The model
+
+With EF Core, data access is performed using a model. A model is made up of entity classes and a context object that represents a session with the database. The context object allows querying and saving data. For more information, see [Creating a Model](xref:core/modeling/index).
 
-With EF Core, data access is performed using a model. A model is made up of entity classes and a context object that represents a session with the database, allowing you to query and save data. See [Creating a Model](xref:core/modeling/index) to learn more.
+EF supports the following model development approaches:
 
-You can generate a model from an existing database, hand code a model to match your database, or use [EF Migrations](xref:core/managing-schemas/migrations/index) to create a database from your model, and then evolve it as your model changes over time.
+* Generate a model from an existing database.
+* Hand code a model to match the database.
+* Use [EF Migrations](xref:core/managing-schemas/migrations/index) to create a database from a model. Migrations allow evolving the database as the model changes.
```

Comment:
This is a bit confusing to me, as Migrations aren't a way to develop/create your model, but rather to create (or sync) a database from a model... It may be worth improving this section and making that clearer ("Once your model has been created, you can use Migrations to keep your database schema in sync with it").